pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+
+There is utility and validity to both approaches. A new PyHC package tiering system is intended to find a "best of both worlds" with the two ideas. Older, out-of-date, unmaintained, or publication-specific code could still have a place for listing and findability, while also allowing nuance between other packages that are more robust, trustworthy, maintained, and work toward the standards-based interpretation of being a PyHC package. 
+
+Further, this tiering system also allows users to get a clearer picture on what each PyHC package has to offer, and the state of the package's condition and development. Creation of a PyHC package tiering system also allows for justification for a myriad of benefits, for example, consideration for funding from a community travel fund, or extra help with improving a standards grouping grade.
+
+# Rationale
+<a name="rationale"></a>
+Decisions for tiering levels, requirements for each tier, and benefits accrued at each tier are based on conversations with the community (bi-annual meetings, telecons, etc.), and are listed here as a starting point for more discussion, likely to be refined in the future. Initially, ideas were presented to the community in a pyramid format. To make the differences between tiers more visible and understandable, it has been transformed into a spreadsheet format. 
+
+# PyHC Package Tiering Specifications
+<a name="specification"></a>
+There are four tiers proposed in this PHEP: Copper, Bronze, Silver, and Gold. These would replace the current structure of "Core Packages", "Other Packages", and "Un-evaluated Packages". See the table below for requirements associated with each tier:
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Self-Evaluation Status | Completed | Completed | Completed | Not Done |
+| PyHC Standard Grades | Mostly green, some yellow | Several yellow, no red | A couple red | N/A |
+| pyOpenSci Review Status | Completed | In progress | Not Started | Not Started |
+| PHEP 3 Compliant? | Yes | Yes | Yes | No |
+| HSSI Metadata Compliant? | Yes | Mostly | Partially | Bare Minimum Met |
+| Software License | Fully compliant and excludes NOSA license | Fully compliant, allows NOSA license | Has non-recommended license (e.g., GPL) | Has no license | 
+| DOI | Required | Required | Required | Not Required |
+| pip installable? | Yes | Yes | Yes | No |
+| conda installable? | Yes | No | No | No |
+
+Descriptions for each heading are as follows:
+   - Self Evaluation Status: indicates whether a package has completed a self evaluation against PyHC's standards
+   - PyHC Standard Grades: indicates status of each standards grouping within a package's self evaluation
+   - pyOpenSci Review Status: indicates status of a pyOpenSci review
```

Comment:
It would be specifically for the PyHC-pyOpenSci pairing review process (checking against both pyOpenSci reqs + PyHC-specific reqs). No link for that, yet. That's part of why I'm trying to get the community to chat about what we'd need to define "yes, fits in with the PyHC" during the pyOpenSci process.

Reply:
Maybe explicitly say "to be defined in the future" or something? It feels a bit weird to be approving a standard that requires something that's not yet in place but I understand we can't do everything at once. Or this could be made more vague of "future collaborations" or something and the PHEP defining the pyOpenSci process would say "modifies PHEP 4 by adding...."

Reply:
That kind of modification plan could work nicely. In that pathway, this PHEP would become the skeleton (most of) the other PHEPs would map to using a stoplight or yes/no system as appropriate.

Reply:
For some items, we can flesh it out here and not wait for another PHEP. Others will need this approach or something similar.

Reply:
Okay, I'm going to modify the wording a bit here based on this feedback.